### ISSUE TITLE
a11y fixes to accordion

### ISSUE THREAD

**Issue by SusannahHarris-HC** [HennepinCounty/ed-hcds-components#1]
The accordion is missing some accessibility features - mainly, when the button is opened, it should say "expanded" or "collapsed".  aria-expanded="false" can be used on each button to ensure that the screen reader reads "collapsed".  you will need additional JS for the aria-expanded="true" attribute to be implemented.  see the examples in the[ P&CL ](https://www.hennepin.us/pattern-components#collapse:~:text=of%20the%20page.-,Collapse,-Back%20to%20main)and [USWDS](https://designsystem.digital.gov/components/accordion/) for guidance.  Also, since the content has an H4, the button headings should be an h3.  See the USWDS site for guidance.  Lastly, please change the name of each accordion header to "Title 1" and so on, the name Header is a little bit confusing. 
- [x] update accordion to include aria-expanded="false" and additional JS for the expanded="true" attribute
- [x] change accordion heading levels to h3
- [x] change the name of the accordion headings to "Title 1" and so on
